examples/ExpoMessaging/ios/ExpoMessaging.xcodeproj/project.pbxproj

```diff
@@ -388,7 +388,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
why was this needed? any clue?

Reply:
Sorry, I committed it unwaveringly and will fix it.